### ISSUE TITLE
User Manager: Changing password and adding host at the same time results in query error

### ISSUE THREAD

**Issue by dmoagx** [sequelpro/sequelpro#2427]
When adding a new host to a user SP will use this query (where password is copied from an existing user):
```sql
CREATE USER 'horst'@'%' IDENTIFIED BY PASSWORD '*27193CCC36970235099CF910EC3128B90A4EB4D4';
```
When changing a pasword the query is:
```sql
SET PASSWORD FOR 'horst'@'%' = PASSWORD('test');
```

However, when both adding a new host and changing the password at the same time the query is:
```sql
CREATE USER 'horst'@'localhost' IDENTIFIED BY PASSWORD 'foobar';
```
This is invalid and will be rejected by MySQL with `Password hash should be a 41-digit hexadecimal number` (or similar).

**Comment by abhibeckert** [comment]
@dmoagx can you confirm this is fixed close the issue?

**Comment by dmoagx** [comment]
This has been included in 1.1.2
